WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemType.kt

```diff
@@ -1,18 +1,27 @@
 package org.wordpress.android.viewmodel.posts
 
 import android.support.annotation.ColorRes
+import org.wordpress.android.fluxc.model.LocalOrRemoteId
 import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.viewmodel.posts.PostListItemIdentifier.LocalPostId
+import org.wordpress.android.viewmodel.posts.PostListItemIdentifier.RemotePostId
 import org.wordpress.android.widgets.PostListButtonType
 
-class PostListItemUiState(
-    val data: PostListItemUiStateData,
-    val actions: List<PostListItemAction>,
-    val onSelected: () -> Unit
-)
+sealed class PostListItemType {
+    // TODO: Can we find a better name for this now that it's a PostListItemType?
+    class PostListItemUiState(
```

Comment:
Maybe something like "ExpandedPost" as a preparation for "CollapsedPost" ... or just LoadedItem/ReadyItem.

Reply:
I am not sure if the `ExpandedPost` will have the meaning we'd want in where it's consumed, I'd like to think on this a bit more before we commit to something if it's OK with you.